docs/multisite.md

```diff
@@ -46,7 +46,20 @@ drush:
     remote: mysite.test
 ```
 
-Then, to refresh your local site, you could run: `blt drupal:sync --site=mysite`.
```

Comment:
Was this removed for a particular reason? I think it needs to stay.

Reply:
Sorry @danepowell That omission wasn't intentional